tools/distpartitioning/convert_partition.py

```diff
@@ -18,6 +18,36 @@ def create_dgl_object(graph_name, num_parts, \
     This function creates dgl objects for a given graph partition, as in function
     arguments. 
 
+    The input schema object is structure as follows: 
```

Comment:
The `schema` argument is a dict, which contains the metadata related to node IDs and edge IDs. It contains two keys: "nid" and "eid", whose value has the following format.

Reply:
modified the text accordingly.